### PR TITLE
 fix margin for mealwrapper and width and height for pic in mealwrapper

### LINKED ISSUE #81: Img saknar höjd+bredd i HTML/jsx
https://github.com/rebecka-oscarsson/lollipop/blob/056ddaa72369d3e75222f3a99a1f7876c0977dc4/src/components/MealWrapper/MealWrapper.jsx#L11-L17

### PR DESCRIPTION
**Pull request by Sarasedaghat8960**
fix margin for meal wrapper and width and height for picture in meal wrapper

### CODE REVIEW

File: src/components/MealWrapper/MealWrapper.jsx [hunk 1]
```diff
@@ -12,6 +12,8 @@ function MealWrapper() {
                         <img
                             className={styles.MealCardPic}
                             src={foodPhoto}
+                            width="180px"
```
Comment (EmilNilsson12) [inline src/components/MealWrapper/MealWrapper.jsx]: ta bort px

det räcker med  t.ex

width="180"

**Review by EmilNilsson12 — CHANGES_REQUESTED** [review]

File: src/components/MealWrapper/MealWrapper.jsx (hunk 1 above)
Comment (Sarasedaghat8960) [reply src/components/MealWrapper/MealWrapper.jsx]: done

**Review by EmilNilsson12 — APPROVED** [review]
Nice!

**Review by axelpahne — APPROVED** [review]
Looks good

**Review by rebecka-oscarsson — APPROVED** [review]

**Pull request merged**